lib/webapp/repo.ex

```diff
@@ -1,3 +1,36 @@
 defmodule Webapp.Repo do
-  use Ecto.Repo, otp_app: :webapp
+  # use Ecto.Repo, otp_app: :webapp
+  @moduledock """
+  Temporary in memory repository.
+  """
+
+  def all(Webapp.User) do
+    [%Webapp.User{
```

Comment:
As mentioned in the commit msg, this is just  a temp "in memory" repo I've created to test the GraphQL stuff. It will be removed after implementing the Ecto Repo.